Integrations/src/main/java/io/deephaven/integrations/learn/Output.java

```diff
@@ -0,0 +1,47 @@
+package io.deephaven.integrations.learn;
+
+import org.jpy.PyObject;
+
+/**
+ * Output specifies how to scatter data from a Python object into a table column.
+ */
+public class Output {
+
+    private final String colName;
+    private final PyObject scatterFunc;
+    private final String type;
+
+    /**
+     * Creates a new Output.
+     *
+     * @param colName       name of new column to store results.
+     * @param scatterFunc   function to scatter the results of a Python object into the table column.
+     * @param type          desired datatype of the new column.
+     */
+    public Output(String colName, PyObject scatterFunc, String type) {
+
+        this.colName = colName;
+        this.scatterFunc = scatterFunc;
+        this.type = type;
+    }
+
+    /**
```

Comment:
See other comments about javadocs.  This needs:
1) To be formated well.  Don't put the comment on the same line as the start of the code
2) Remove "getter" from the javadoc.